src/storage/mod.rs

```diff
@@ -993,17 +988,22 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
         cf: String,
         key: Vec<u8>,
         value: Vec<u8>,
+        ttl: u64,
         callback: Callback<()>,
     ) -> Result<()> {
         check_key_size!(Some(&key).into_iter(), self.max_key_size, callback);
-
+        let mut m = Modify::Put(Self::rawkv_cf(&cf)?, Key::from_encoded(key), value);
+        if self.enable_ttl {
```

Comment:
Do we need to add another key space (prefix) when `enable_ttl` ? Or we shall add in `RawKVClient`?

Reply:
Have discussed with @BusyJay before, adding the key prefix is wasteful. If user wants to use keyspace, it's the user's duty to write the key with the prefix.